web/html/src/manager/admin/payg/payg.renderer.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+
+import { RolesProvider } from "core/auth/roles-context";
+import SpaRenderer from "core/spa/spa-renderer";
+
+import { MessagesContainer } from "components/toastr";
+
+import Pyag from "./payg";
+
+type RendererProps = {
+  payg?: string;
+  wasFreshlyCreatedMessage?: string;
+};
+
+export const renderer = (id: string, { payg, wasFreshlyCreatedMessage }: RendererProps = {}) => {
+  let paygJson: any = {};
+  try {
+    paygJson = JSON.parse(payg || "");
+  } catch (error) {
+    console.log(error);
+  }
+
+  SpaRenderer.renderNavigationReact(
+    <RolesProvider>
+      <MessagesContainer />
+      <Pyag payg={paygJson} {...(wasFreshlyCreatedMessage && { wasFreshlyCreatedMessage })} />
```

Comment:
The logic around `wasFreshlyCreatedMessage` looks a bit confusing, what's the intent here?

Reply:
I have simplified the code in this, and I have tested and looks like it is working. Thanks